### ISSUE TITLE
Accessibility issues reported by user

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/righttoknow#977]
From a concerned user:
> the main heading across the website is white text on a pale blue ground. The navigation is the same white on a slightly darker teal. 
> These fail the simplest accessibility requirement, an adequate contrast between the text and background. what makes it worse is that the test takes 10 seconds to run and the colour choices equally as long. This is a 2 minute fix yet you persist. 
> What led you to make such a poor decision and make what is otherwise a very useful and important website so inaccessible to a sizeable proportion of the community?

--
Source: https://mail.missiveapp.com/#team_unassigned/8555ecd2-4304-41e5-ab2d-17e392eab5df_team_unassigned/conversations/34a6f426-47c2-4ce7-8d38-8f531307382b

**Comment by benrfairless** [comment]
<p>I&apos;ve run a scan in BrowserStack, quite a number of serious issues that we should probably look at.

Putting this on the Team Planning board agenda for Feb at this stage. I've let the user who reached out to us know that, if they are able to submit a PR, we are happy to review it sooner

<details><summary>BrowserStack Details</summary>
</p><p>Test details: Open <a href="https://accessibility.browserstack.com/site-scanner/projects/www.righttoknow.org.au-1/builds?thBuildId=wil5tndcpidnhxgbs8znvj50jqkmumpzlo0mg0oc">URL</a> on Browserstack</p><br />Summary details<table><tr><th>Item&nbsp;&nbsp;</th><th>Data&nbsp;&nbsp;</th></tr><tr><td>Report Name&nbsp;&nbsp;</td><td>www.righttoknow.org.au-1&nbsp;&nbsp;</td></tr><tr><td>Type of Test&nbsp;&nbsp;</td><td>Website Scan&nbsp;&nbsp;</td></tr><tr><td>WCAG version&nbsp;&nbsp;</td><td>WCAG 2.1 AA&nbsp;&nbsp;</td></tr><tr><td>Issue count&nbsp;&nbsp;</td><td>41&nbsp;&nbsp;</td></tr><tr><td>Scan timestamp&nbsp;&nbsp;</td><td>2025-12-09T05:50:46.689Z&nbsp;&nbsp;</td></tr></table>Issue count with breakdown by severity<table><tr><th>Item&nbsp;&nbsp;</th><th>Data&nbsp;&nbsp;</th></tr><tr><td>Critical&nbsp;&nbsp;</td><td>1&nbsp;&nbsp;</td></tr><tr><td>Serious&nbsp;&nbsp;</td><td>33&nbsp;&nbsp;</td></tr><tr><td>Moderate&nbsp;&nbsp;</td><td>6&nbsp;&nbsp;</td></tr><tr><td>Minor&nbsp;&nbsp;</td><td>1&nbsp;&nbsp;</td></tr></table>
</details>

**Comment by jrmadsen67** [comment]
Hi @benrfairless !

I have a BroswerStack account at the email you have for me - if that report is still valid, can you give me permissions to see it?

My thought is to try to address all (or as many as I can) of the accessibilty issues with this task, if it makes sense

My first thought here is to replace the teal with the maroon color from here https://donate.oaf.org.au/ that seems to be our primary color

**Comment by benrfairless** [comment]
Hi @jrmadsen67,

Thanks for taking a look at this 🥇 

> I have a BroswerStack account at the email you have for me - if that report is still valid, can you give me permissions to see it?

I've made the report public - see [here](https://accessibility.browserstack.com/public_report?type=scanner&token=tok_ae9b0fbf7e853ac2fa824716bdce49c20e98671d95ad8fc9f6aeb56754901a14354a7e2bc42dbed80ea96168406c48c037636dfe4828216ee0295a4d3b45076e). Hopefully that helps.

> My first thought here is to replace the teal with the maroon color from here https://donate.oaf.org.au/ that seems to be our primary color

We tend to use a different colour scheme for each of our services so people can distinguish them - see [TheyVoteForYou](https://theyvoteforyou.org.au/) and [Planning Alerts](https://www.planningalerts.org.au/) for instance, they look completely different. 

You can also see [here](https://alaveteli.org/deployments/) for a list of services around the world that also use Alaveteli (for some inspiration), as this is basically just an Alaveteli theme anyway.

**Comment by benrfairless** [comment]
Also worth pointing out - CSS from this theme is injected into [our core](https://github.com/openaustralia/alaveteli) on launch, replacing any defaults that might exist. But if we don't replace something, then the default from the core takes effect.

**Comment by Br3nda** [comment]
Broke this out into three sub-issues (#1077, #1078, #1079) after digging into the actual cause.

**Tools used, in case useful for future accessibility work:**

- **BrowserStack's accessibility scanner** (already linked above) - tests against WCAG 2.1 AA, needs a BrowserStack account to run new scans, but the report link is shareable/public once generated.
- **[pa11y](https://github.com/pa11y/pa11y)** - open source CLI, no account needed, built on axe-core + HTML_CodeSniffer. `npx pa11y <url> --reporter json` gives a full itemised report. Good for a quick independent check or wiring into CI later (`pa11y-ci` can fail a build on regressions). We used this to try to get a second, independent read on the site.
- Also worth knowing about: **Lighthouse** (`lighthouse <url> --only-categories=accessibility`, ships with Chrome DevTools, also axe-core based) and **WAVE** (wave.webaim.org, good for quick manual spot-checks in browser).
- For the two contrast sub-issues (#1077, #1078) specifically, the numbers came from computing real WCAG contrast ratios against the actual Sass colour variables in `righttoknow`'s `_settings.scss`/`custom.scss` (relative luminance formula, per the WCAG 2.1 spec), not just eyeballing it - see those issues for the exact figures.

None of these replace a screen-reader/keyboard pass by an actual person, but they're a solid first sweep and easy for a contributor to re-run locally to check their own fix.
